backend/tests/test_contact.py

```diff
@@ -56,3 +56,85 @@ def test_delete_a_contact(self):
         # deleting a contact that does not exist
         response = self.client.delete("/contact/{}".format(c_id))
         self.assertEqual(response.status_code, 404)
+
+    # testing creating a contact
+    def test_create_a_contact(self):
+        dummy_ct_id = uuid.uuid4()
+        dummy_ct = ContactType(
+            id=dummy_ct_id,
+            hex_colour="#ffffff",
+            type="dummy type",
+            description="dummy description",
+            contacts=[],
+        )
+        db.session.add(dummy_ct)
+        db.session.commit()
+
+        # creating a contact with valid inputs
+        response = self.client.post(
+            "/contact",
+            json={
+                "name": "dummy name",
+                "email": "dummy email",
+                "secondary_email": "dummy secondary email",
+                "cellphone": "dummy cell phone",
+                "role": "dummy role",
+                "organization": "dummy organization",
+                "neighbourhood": "dummy neighbourhood",
+                "contact_type": dummy_ct_id,
+            },
+        )
+        self.assertEqual(response.status_code, 200)
+        c = json.loads(response.get_data(as_text=True))
+
+        # check that the contact type has been properly added
+        contact_type = Contact.query.filter_by(id=c["id"]).first()
```

Comment:
@dgiltz Normally in unit tests you would want to make what you're verifying in your test as clear as possible. Meaning people are able to know what you're expecting and what the actual returned value is, what should make it pass and what should make it fail.

Here though you are querying the contact you created, it is not clear as to what should make it fail and what should make it pass. Here if the contact creation were to fail, `.first()` would simply return a `None`, which wouldn't really notify the developer that the contact creation failed.

Instead, you can do a very surface level test with `assertIsNotNone` to verify that you are able to query the contact you just created with the id from the response. This way if `.first()` were to return `None`, the assertion would fail the test.